documentation/src/main/asciidoc/reference/mapper-orm-bridge-propertybridge.asciidoc

```diff
@@ -263,13 +263,46 @@ include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/bridge/proper
 include::components/incubating-warning.asciidoc[]
 
 The context passed to the property binder's `bind` method
-exposes a `bridgedElement()` method that gives access to metadata about the property being bound,
-in particular its name and type.
+exposes a `bridgedElement()` method that gives access to metadata about the property being bound.
 
-The metadata can also be used to inspect the type of the property in details:
+The metadata can be used to inspect the property in details:
 
+* Getting the name of the property.
+* Checking the type of the property.
 * Getting accessors to properties.
 * Detecting properties with markers.
 Markers are applied by specific annotations carrying a `@MarkerBinding` meta-annotation.
 
 See the javadoc for more information.
+
+Below is an example of the simplest use of this metadata,
+getting the property name and using it as a field name.
+
+.Naming a field after the property being bound in a `PropertyBinder`
+====
+[source, JAVA, indent=0, subs="+callouts"]
+----
+include::{sourcedir}/org/hibernate/search/documentation/mapper/orm/bridge/propertybridge/bridgedelement/InvoiceLineItemsSummaryBinder.java[tags=include]
+----
+<1> Use the binding context to get the bridged element.
+<2> Use the name of the property as the name of a declared field.
```

Comment:
It is almost clear, but I would write something more explicit, like:
> Use the name of the property to get the access to the `objectField`

Reply:
That's not right, though. We're not really "getting access" to the field, we're defining an index field. It doesn't exist before this line.

I'll clarify that I mean "index" field.